extensions/jpa/runtime/pom.xml

```diff
@@ -44,14 +49,14 @@
             <groupId>org.apache.camel.quarkus</groupId>
             <artifactId>camel-quarkus-core</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.camel.quarkus</groupId>
-            <artifactId>camel-quarkus-support-spring</artifactId>
-        </dependency>
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-jpa</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-tx</artifactId>
```

Comment:
Could maybe scope as `provided`?

Not sure if it'd be possible eventually to remove this entirely. It'd be nice if we could.

Reply:
`java.lang.ClassNotFoundException: org.springframework.transaction.support.TransactionTemplate` is throws during static analyst if we add `provided` scope.

Reply:
Finnaly we remove it! and add fake `PlatformTransactionManager` and `TransactionTemplate` to make it work with graalvm native bulding.